airgun/entities/sync_templates.py

```diff
@@ -0,0 +1,53 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep
+from airgun.navigation import navigator
+from airgun.views.template import SyncTemplatesView
+
+
+class SyncTemplatesEntity(BaseEntity):
+    endpoint_path = '/template_syncs'
+
+    def import_templates(self, values):
+        """Import Templates entity"""
+        view = self.navigate_to(self, 'import')
+        view.fill(values)
+        view.submit.click()
+
+    def export_templates(self, values):
+        """Export Templates entity"""
+        view = self.navigate_to(self, 'export')
+        view.fill(values)
+        view.submit.click()
+
+
+@navigator.register(SyncTemplatesEntity, 'All')
```

Comment:
While it's true that almost all of our entities register "All" endpoint, it's not strictly required. I always understood "All" as "list of all entities", and this description doesn't really fit this page. I would rather name it "Main", or "ImportExport", or "Sync".

Reply:
TBH, I was planning to remove this altogether cause, the endpoint_path = '/template_syncs' directly takes me to that page. :)